### PR TITLE
update core

### PR DESCRIPTION
**Pull request by rwmcintosh**
A new core was released for 11.2 PK-Sim and MoBi.

### CODE REVIEW

File: appveyor-nightly.yml [hunk 1]
```diff
@@ -43,9 +43,13 @@ before_build:
   - rake "prepare_for_build[%APPVEYOR_BUILD_VERSION%]"
 
 build_script:
+  - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
   - travis-tool.sh install_deps
   - travis-tool.sh r_binary_install curl
-  - Rscript install_dependencies.R
+  - Rscript -e "install.packages('covr', repos = 'http://cran.us.r-project.org')"
+  - Rscript -e "install.packages('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip', repos = NULL, type = 'binary')"
```
Comment (msevestre) [inline appveyor-nightly.yml:50]: This is probably all wrong on develop? There is this new install dep scripts

**Review by msevestre — COMMENTED** [review]
I don't think we want to revert all the changes in appveyor file on develop. Is this by mistake?

File: appveyor-nightly.yml [hunk 2]
```diff
@@ -61,8 +65,6 @@ test_script:
 after_test:
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, ubuntu18]"
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, centOS7]"
-  - rake "download_portable[develop]"
```
Comment (rwmcintosh) [inline appveyor-nightly.yml:64]: Should this also stay? It's new on Develop too for creating a standalone OSPSuite-R?

**Comment by rwmcintosh** [comment]
> I don't think we want to revert all the changes in appveyor file on develop. Is this by mistake?

All those appveyor file changes are reverted. Just the binary and config files left.

**Review by PavelBal — APPROVED** [review]

**Pull request merged**